src/BenchmarkDotNet/Jobs/Argument.cs

```diff
@@ -33,6 +33,20 @@ public MonoArgument(string value)
     [PublicAPI]
     public class MsBuildArgument : Argument
     {
-        public MsBuildArgument(string value) => TextRepresentation = value;
+
+        public MsBuildArgument(string value) => TextRepresentation = EscapeBuildArguments(value);
+
+        /// <summary>
+        /// Escapes the \ with \\ when \ is part of the build argument.
+        /// <see href="https://github.com/dotnet/BenchmarkDotNet/issues/1536">Fixes issue 1536</see>
+        /// </summary>
+        /// <param name="args"></param>
+        /// <returns></returns>
+        public string EscapeBuildArguments(string args)
```

Comment:
could you please make this method internal and add some simple unit test for it?

Reply:
Marked the method as internal instead of public